### DISCUSSION TITLE
Would you consider to implement bracketSameLine ?

### DISCUSSION THREAD

**Discussion by nhuray** [rome/tools#3919] [General]
Hi community,

I know Rome is opinionated about formatting and I agree it should offer minimum configuration. 

However as discussed in the following [topic](https://github.com/rome/tools/discussions/2611#discussioncomment-2830786) about Prettier Compatibility, an important incentive to move from Prettier to Rome is to ensure the amount of changes would be sufficiently small (what @NicholasLYang refers to [Pull Request Compatibility](https://github.com/rome/tools/discussions/2611#discussioncomment-2830325))

Recently, Rome implemented [semicolons](https://github.com/rome/tools/pull/3702) option which was something requested by the community. That decision clearly lower the barrier for the community to migrate to Rome.

I'm wondering considering the [bracketLine](https://prettier.io/docs/en/options.html#bracket-line) option is also [heavily](https://github.com/search?q=bracketSameLine&type=code) used, if it might be a good candidate for implementation.

Any feedback on that discussion is appreciated.

**Comment by ematipico** [comment]
> I'm wondering considering the [bracketLine](https://prettier.io/docs/en/options.html#bracket-line) option is also [heavily](https://github.com/search?q=bracketSameLine&type=code) used, if it might be a good candidate for implementation.

I am wondering if you could refine your search. I opened the link where you suggested that the option is heavily used, but many results point to forks of the `prettier` repository, and many code snippets belong to `prettier`'s test suite.

**Comment by nhuray** [comment]
@ematipico sorry for the late answer !

I made a new research in GitHub [advanced search](https://github.com/search/advanced?q=bracketSameLine+filename%3Apackage.json+filename%3A.prettierrc+filename%3A.prettierrc.json+filename%3A.prettierrc.yml+filename%3A.prettierrc.yaml+filename%3A.prettierrc.json5+filename%3A.prettierrc.js+filename%3A.prettierrc.cjs+filename%3Aprettier.config.js+filename%3Aprettier.config.cjs+filename%3A%22.prettierrc.toml+language%3AJSON+language%3AJSON%22&type=Code) to find the term `bracketSameLine` in the [prettier config files](https://prettier.io/docs/en/configuration.html).

I found around 50k occurrences (JS + JSON):

![StaticShot_23-12-2022_14-49-33](https://user-images.githubusercontent.com/958983/209347711-8183d50f-2b93-44e8-aa8a-316190f50623.png)


For some reasons Github does not return the results so I can't really filter out more to find when bracketSameLine is `true`.

**Let me know if you have a better way to refine that research.**
